DependencyInjection/LiipMonitorExtension.php

```diff
@@ -255,6 +253,26 @@ private function configureDoctrineMigrationsCheck(ContainerBuilder $container, a
         }
     }
 
+    private function configureMailer(ContainerBuilder $container, LoaderInterface $loader, array $config)
+    {
+        if (false === $config['mailer']['enabled']) {
+            return;
+        }
+
+        try {
+            $mailerDefinition = $container->findDefinition('mailer');
```

Comment:
Hi @Baachi , @kbond 

I am unable to make the bundle work when I enable the mailer configuration.

I am using Symfony 4.4 (symfony/mailer 4.4, symfony/mailgun-mailer 4.4), LiipMonitorBundle 2.11.1

Regarding the [Symfony documentation](https://symfony.com/doc/4.3/bundles/extension.html#using-the-load-method)
> In the load() method, all services and parameters related to this extension will be loaded. This method doesn't get the actual container instance, but a copy. This container only has the parameters from the actual container. After loading the services and parameters, the copy will be merged into the actual container, to ensure all services and parameters are also added to the actual container.

The container copy won't never have any mailer definition, so the exception is always raise.

Does it need a CompilerPass to load the mailer configuration instead?

Thanks!

Reply:
Good catch @ixarlie, it looks like we should just be setting the config parameters in the extension and like you suggest, have a compiler pass to create the service.

@Baachi, are you able to take a look?

Reply:
Hi @kbond , @Baachi  finally I've opened a new pull request addressing this issue. https://github.com/liip/LiipMonitorBundle/pull/227